clients/mtaclient/fakes/fake_mta_client_operations.go

```diff
@@ -74,9 +74,10 @@ type FakeMtaClientOperations struct {
 		result1 []*models.Log
 		result2 error
 	}
-	GetMtaOperationsStub        func(last *int64, status []string) ([]*models.Operation, error)
+	GetMtaOperationsStub        func(mtaId *string, last *int64, status []string) ([]*models.Operation, error)
```

Comment:
How you added this? Generated it?
I am asking because the fakes are generated and thus not good to be touched.

Reply:
I didnt generated it, because I didnt know that

Reply:
Is there any description somewhere on how to generate fakes?

Reply:
We are using this tool:
https://github.com/maxbrunsfeld/counterfeiter